README.md

```diff
@@ -66,3 +66,27 @@ We use PR reviews to approve or reject, comment on, and request further iteratio
 - DRY
 - Readability & Transparency: Code as language
 - Favor explicitness over defensiveness
+
+## Deploying Using AWS SAM
+
+We are using AWS SAM to build the rikolti lambda applications and deploy to AWS. Following are proposed steps for building and deploying using SAM:
+
+Make sure you [have SAM CLI installed](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html).
+
+From the rikolti directory, which contains `template.yaml`, build the serverless applications:
+
+```
+sam build
+```
+
+> **NOTES:**
+>
+> There is a [troposphere](https://troposphere.readthedocs.io/en/latest/quick_start.html) script stub named `create_sam_template.py` checked into the repo. This script generates `template.yaml`, but we decided not to use it for now since the template is simple and we don't need to introduce another layer of tooling at this point. We might use it in the future to generate templates for different environments and such.
+
+> We will eventually want to use `sam build` with the `--use-container` option for compiling the `lxml` dependency for the `metadata_mapper`, as lxml needs to be natively compiled, i.e. has C bindings and is not pure python.
```

Comment:
Is there a blocker on figuring this out? Shouldn't we just figure it out now while we're in the midst of all this SAM stuff? Sorry - I guess what I meant this morning was we didn't need to do it together on the call, but we should probably get this sorted now? Unless you just want to make a new card to add to your stack for next sprint?

Reply:
Oh yeah I totally misunderstood what we were saying yesterday! I will figure this out and add it to the README before we merge this in (:

Reply:
@amywieliczka OK it looks like it's pretty straightforward to use `sam build --use-container`. I updated the README with instructions for doing so.

Reply:
wow, you don't even have to provide a container. that's some magic